control/grpc.py

```diff
@@ -56,6 +56,43 @@ def group_id(self, request: monitor_pb2.group_id_req, context = None) -> Empty:
         self.set_group_id(request.id)
         return Empty()
 
+class HostInfo:
```

Comment:
```suggestion
class SubsystemHostAuth:
```
As this operates at subsystem level, not host, and it's about authentication & authorization.

BTW, don't we already have any existing subsystem helper class to add this functionality to?

Reply:
We have one for namespaces and one for listeners. I didn't want to change too much in places I didn't have to. We might choose to merge some of these in the future. I'll rename the class as you suggested.